CMakeLists.txt

```diff
@@ -237,8 +243,17 @@ endif()
 add_subdirectory(external/WALi-OpenNWA)
 include_directories(external/WALi-OpenNWA/Source/wali/include)
 
+# Set up clang-tidy to run during PhASAR's compilation to indicate code smells
+if (PHASAR_ENABLE_CLANG_TIDY_DURING_BUILD)
```

Comment:
Just as food for thought, in our project we don't run tidy all the time during normal compilation but have a specific cmake target that runs tidy over the code base (`ninja tidy-foo`). I found this a bit more convenient as tidy can sometimes take quite a bit of time to complete.